src/obr/signac_wrapper/submit.py

```diff
@@ -0,0 +1,74 @@
+import shutil
+import os
+import logging
+
+from pathlib import Path
+from signac.job import Job
+from typing import Union
+
+
+def submit_impl(
+    project,
+    jobs,
+    operations: list[str],
+    template: Union[str, None],
+    account: Union[str, None],
+    partition: Union[str, None],
+    pretend: bool,
+    bundling_key: Union[str, None],
+    scheduler_args: str,
+):
+    template_target_path = Path(project.path) / "templates/script.sh"
+    template_src_path = Path(template)
+
+    if not template_src_path.exists():
+        raise FileNotFoundError(template)
+
+    if template and template_target_path.exists():
+        shutil.rmtree(template_target_path.parent)
+
+    if template:
+        os.makedirs(template_target_path.parent, exist_ok=True)
+        shutil.copyfile(template_src_path, template_target_path)
+
+    project._entrypoint = {"executable": "", "path": "obr"}
```

Comment:
Since this is a private variable, I would prefer to add a setter for the entrypoint, just for best practice's sake.